### PR TITLE
CONCF-1074 | Fix Styles with scroll bars

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: front/styles/main/component/_curated-content-editor.scss [hunk 1]
```diff
@@ -276,15 +265,11 @@ $search-results-load-more-button-width: 100vw - 2 * $search-results-horizontal-p
...
-	height: $row-height;
-	text-align: right;
-	width: calc(8vw - #{$item-padding-horizontal});
+	flex: 0 20px;
 }
 
-.arrow-up, .arrow-down {
-	height: $row-height / 2;
+.arrow {
+	height: $row-height / 3;
```
Comment (hakubo) [inline front/styles/main/component/_curated-content-editor.scss]: those will be reworked anyway, so I am leaving it as is for now

**Comment by rogatty** [comment]
LGTM but it needs some regression testing, especially on iPhones (`cursor: pointer` magic).

File: front/templates/main/components/curated-content-editor-item-form.hbs [hunk 2]
```diff
@@ -43,19 +43,13 @@
...
-						{{else}}
-							<li class="message {{if suggestionsError errorClass}}">{{searchSuggestionsMessage}}</li>
-						{{/if}}
-						</ul>
-					</div>
-				{{/if}}
+				<ul class="title-suggestions-container {{unless searchSuggestionsVisible 'hidden'}}">
+					{{#each searchSuggestionsResult as |item|}}
+						<li {{action 'setTitle' item.title on='mouseDown'}} class="needsclick">{{item.title}}</li>
+					{{else}}
```
Comment (tomasznapieralski) [inline front/templates/main/components/curated-content-editor-item-form.hbs]: When it goes to that block?

Comment (hakubo) [inline front/templates/main/components/curated-content-editor-item-form.hbs]: http://handlebarsjs.com/builtin_helpers.html#iteration

When `searchSuggestionsResult` is empty

Comment (tomasznapieralski) [inline front/templates/main/components/curated-content-editor-item-form.hbs]: ok

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**